tests/xcodeproj/test.swift

```diff
@@ -2,11 +2,7 @@ import XCTest
 
 class SwiftTests : XCTestCase {
   func testPasses() {
-      let envvarvalue = ProcessInfo.processInfo.environment["test_envvar_key"]
```

Comment:
why did you get rid of the tests for the args / env var?

Reply:
See comment below. The functionality being added in this PR can be tested by doing `git diff` on the xcodeproject file and ensuring it hasn't changed.